executing/executing.py

```diff
@@ -534,17 +534,20 @@ def compile_similar_to(source, matching_code):
 
 sentinel = 'io8urthglkjdghvljusketgIYRFYUVGHFRTBGVHKGF78678957647698'
 
+def is_rewritten_by_pytest(code):
+    return any(
+        bc.opname.startswith("LOAD_") and isinstance(bc.argval,str) and bc.argval.startswith("@py_assert")
```

Comment:
```suggestion
        isinstance(bc.argval,str) and bc.argval.startswith("@py")
```

Does this also work?